charts/stfc-cloud-vm/values.yaml

```diff
@@ -0,0 +1,65 @@
+victoria-metrics-cluster:
+  vmselect:
+    replicaCount: 5
+
+    ingress:
+      enabled: true
+      ingressClassName: nginx
+      annotations:
+        kubernetes.io/ingress.class: "nginx"
+        cert-manager.io/cluster-issuer: "self-signed"
```

Comment:
Does the cert always have to be self-signed?

Reply:
No? These are default values which can be overridden by the user